psh/test-gibber.py

```diff
@@ -0,0 +1,80 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh gibbering input testing
+#
+# Copyright 2021, 2022 Phoenix Systems
+# Author: Damian Loewnau
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+
+import string
+import psh.tools.psh as psh
+import trunner.config as config
+
+
+EXPECTED = 'Unknown command!'
+EOT = '\x04'
+
+
+def assert_letters(p):
+    msg = 'Wrong output for lowercase letters'
+    psh.assert_cmd(p, string.ascii_lowercase, EXPECTED, msg)
+
+    msg = 'Wrong output for uppercase letters'
+    psh.assert_cmd(p, string.ascii_uppercase, EXPECTED, msg)
+
+    msg = 'Wrong output for all ascii letters'
+    psh.assert_cmd(p, string.ascii_letters, EXPECTED, msg)
+
+
+def assert_digits(p):
+    msg = 'Wrong output for all digits'
+    psh.assert_cmd(p, string.digits, EXPECTED, msg)
+
+
+def assert_punctuation(p):
+    msg = 'Wrong output for punctuation characters'
+    psh.assert_cmd(p, string.punctuation, EXPECTED, msg)
```

Comment:
Mabye do all printables in single prompt?

Reply:
Yeah, please check the latest force-push